diagnostics/model_accuracy/model-run.json

```diff
@@ -0,0 +1,29 @@
+[
+
+  {
+    "model_name": "Inception_v4",
+    "accuracy" : "0.802",
+    "model_type": "Image Recognition",
+    "cmd": "OMP_NUM_THREADS=28 KMP_AFFINITY=granularity=fine,compact,1,0, python eval_image_classifier.py --alsologtostderr --checkpoint_path=/nfs/fm/disks/aipg_trained_dataset/ngraph_tensorflow/fully_trained/inception4_slim/inception_v4.ckpt --dataset_dir=/mnt/data/TF_ImageNet_latest ---dataset_name=imagenet --dataset_split_name=validation --model_name=inception_v4"
+  },
+
+  {
+    "model_name": "MobileNet_v1",
+    "accuracy" : "0.709",
+    "model_type": "Image Recognition",
+    "cmd": "OMP_NUM_THREADS=28 KMP_AFFINITY=granularity=fine,compact,1,0, python eval_image_classifier.py --alsologtostderr --checkpoint_path=/nfs/fm/disks/aipg_trained_dataset/ngraph_tensorflow/fully_trained/mobilenet_v1/mobilenet_v1_1.0_224.ckpt --dataset_dir=/mnt/data/TF_ImageNet_latest ---dataset_name=imagenet --dataset_split_name=validation --model_name=mobilenet_v1"
+  },
+
+  {
+    "model_name": "ResNet50_v1",
+    "accuracy" : "0.752",
+    "model_type": "Image Recognition",
+    "cmd": "OMP_NUM_THREADS=28 KMP_AFFINITY=granularity=fine,compact,1,0, python eval_image_classifier.py --alsologtostderr --checkpoint_path=/nfs/fm/disks/aipg_trained_dataset/ngraph_tensorflow/fully_trained/resnet50_v1_slim/resnet_v1_50.ckpt --dataset_dir=/mnt/data/TF_ImageNet_latest --dataset_name=imagenet --dataset_split_name=validation --model_name=resnet_v1_50 --labels_offset=1"
+  },
+
+  {
+    "model_name": "SSD-MobileNet_v1",
```

Comment:
There is no "accuracy" for the SSD-MobileNet_v1 model, but there wasn't in the original script either.  @avijit-nervana and @sindhu-nervana : how do you want to handle this?  Should an accuracy be added, or the SSD-MobileNet_v1 model be removed from the JSON file, or simply leave as is (which may cause odd exceptions when the script is run with SSD-MobileNet_v1)?

Reply:
@crlishka I was not able to run SSD-MobileNet successfully when i developed this script, so did not mention the model name in the list of available ones in the help here https://github.com/tensorflow/ngraph-bridge/blob/master/diagnostics/model_accuracy/verify_inference_model.py#L132. It needs to be debugged further as we continue to enable more models. Also, something to keep in mind we might have to keep updating the regex and any other dependencies/patches to apply in this script as we enable more models. You could add accuracy and keep it in json for now or remove it entirely, really up to you.

Reply:
To keep things simple and remove the SSD-MobileNet model then.  If it is not running, and we do not have an accuracy target, it is probably better to remove it.

Reply:
@crlishka , @sindhu-nervana , I'm removing the entire entry ofSSD-MobileNet  in the json file.